website/docs/r/nsxt_network_dhcp.html.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_network-dhcp"
+sidebar_current: "docs-vcd-resource-nsxt-network-dhcp"
+description: |-
+  Provides a resource to manage DHCP pools for NSX-T Org VDC Routed network.
+---
+
+# vcd\_nsxv\_network\_dhcp
+
+Provides a resource to manage DHCP pools for NSX-T Org VDC Routed network.
+
+Supported in provider *v3.2+* and VCD 10.1+ with NSX-T backed VDCs.
+
+## Specific usage notes
+
+**DHCP pool support** for NSX-T Routed networks is **limited** by the API in the following ways:
+
+* DHCP pools can only be added, but not updated/removed one by one
+
+* VCD 10.2+ allows to remove all DHCP pools at once (terraform destroy/resource removal)
+
+
+* VCD 10.1 **does not allow to remove DHCP pools** after they are created without removing parent network
```

Comment:
Wow, pretty hard limitation.

Reply:
Indeed. However even in 10.2 UI does not allow to remove DHCP pools.